applications/vessel-history/src/features/i18n/i18nDate.tsx

```diff
@@ -71,3 +71,7 @@ export const I18nSpecialDate = ({ date, format = DateTime.DATE_MED }: TMTDate) =
 }
 
 export default I18nDate
+
+export const I18nDateAsString = (date: string , format = DateTime.DATE_MED) => {
+  return useI18nDate(date, format)
+}
```

Comment:
We don't need this function, instead we should just call `useI18nDate(date, format)` from the component
```suggestion

```

Reply:
not work, inside a conditional use that function breaks

Reply:
You can use `formatI18nDate` as[ I suggested below](https://github.com/GlobalFishingWatch/frontend/pull/996/files#r717548433) and it does work

Reply:
yes I saw it then